cpp/modmesh/view/RManager.cpp

```diff
@@ -117,57 +117,46 @@ void RManager::setUpMenu()
     // NOTE: All menus need to be populated or Windows may crash with
     // "exited with code -1073740791".  The reason is not yet clarified.
 
+    m_fileMenu = m_mainWindow->menuBar()->addMenu(QString("File"));
     {
-        m_fileMenu = m_mainWindow->menuBar()->addMenu(QString("File"));
-
-        {
-            auto * action = new RAction(
+        m_fileMenu->addAction(
+            new RAction(
                 QString("New file"),
                 QString("Create new file"),
                 []()
                 {
                     qDebug() << "This is only a demo: Create new file!";
-                });
-            m_fileMenu->addAction(action);
-        }
+                }));
 
-        m_eulerMenu = m_mainWindow->menuBar()->addMenu(QString("Euler"));
+        m_fileMenu->addAction(new RAction(
+            QString("Parameters"),
+            QString("Runtime parameters"),
+            []()
+            {
+                static int64_t int64V = 5566;
+                static double doubleV = 77.88;
+                auto params = createParameters();
+                addParam(params, "global.a.b.int64_foo", &int64V);
```

Comment:
what's the purpose of this?

Reply:
Good catch.  It's experimental code @buganini added to test interoperability with PUI parameter transition: https://discord.com/channels/730297880140578906/1248934211948253224 .  It doesn't work right now.  We will fix it once needing to rectify the handling of GUI parameters.